apps/backend/src/modules/categories/category.controller.ts

```diff
@@ -42,6 +43,56 @@ class CategoryController extends BaseController {
 			method: "GET",
 			path: CategoriesApiPath.ROOT,
 		});
+
+		this.addRoute({
+			handler: (options) =>
+				this.findByIds(
+					options as APIHandlerOptions<{
+						query: CategoriesGetByIdsRequestDto;
+					}>,
+				),
+			method: "GET",
+			path: CategoriesApiPath.GET_BY_ID,
+		});
+	}
+
+	/**
+	 * @swagger
+	 * /categories/get-by-id:
```

Comment:
Should be the same route as for get all. If no query - return all, if there's a query - return specific ones